CleverPush/Source/CPAppBannerModuleInstance.m

```diff
@@ -196,7 +196,17 @@ - (void)getBanners:(NSString*)channelId bannerId:(NSString*)bannerId notificatio
 
     NSMutableURLRequest* request = [[CleverPushHTTPClient sharedClient] requestWithMethod:HTTP_GET path:bannersPath];
     [CleverPush enqueueRequest:request onSuccess:^(NSDictionary* result) {
-        NSArray *jsonBanners = [result objectForKey:@"banners"];
+        NSMutableArray *jsonBanners = [[NSMutableArray alloc]init];
+        if(categoryId != nil && ![categoryId isEqualToString:@""]) {
```

Comment:
space after if

Reply:
Also, please do the whole filtering in an extra function to keep things clean here.